examples/viewport-animation/src/app.js

```diff
@@ -28,7 +28,9 @@ export default class App extends Component {
       latitude,
       zoom: 11,
       transitionInterpolator: new FlyToInterpolator(),
-      transitionDuration: 3000
+      transitionDuration: 'auto',
+      transitionSpeed: 2,
```

Comment:
My understanding is that this option is specific to `FlyToInterpolator`. Move this option into the `FlyToInterpolator` constructor?

Reply:
Updated.